homeassistant/components/anova_sous_vide/__init__.py

```diff
@@ -0,0 +1,40 @@
+"""The Anova integration."""
+from __future__ import annotations
+
+from anova_wifi import AnovaOffline, AnovaPrecisionCooker, AnovaPrecisionCookerSensor
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN
+from .coordinator import AnovaCoordinator
+
+PLATFORMS = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Anova from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
```

Comment:
Don't set this until after you know the integration is going to actually setup (ie, passed though anything that can raise ConfigEntryNotReady)